src/federated.c

```diff
@@ -225,21 +230,29 @@ void FederatedConnectionBundle_handle_tagged_msg(FederatedConnectionBundle *self
       ret = sched->schedule_at_locked(sched, &event);
       switch (ret) {
       case LF_AFTER_STOP_TAG:
-        LF_WARN(FED, "Tried scheduling event after stop tag. Dropping\n");
+        LF_WARN(FED, "Tried scheduling event after stop tag. Dropping");
         break;
       case LF_PAST_TAG:
-        LF_WARN(FED, "Tried scheduling event to a past tag. Dropping\n");
+        LF_ERR(FED, "Safe-to-process violation! Tried scheduling event to a past tag. Handling now instead!");
```

Comment:
If we receive an event destined for a past tag, then instead of dropping it, we handle it ASAP. In the future we will add support for a handler for this case, just like the deadlines